pkg/build/golang/docker.go

```diff
@@ -192,7 +192,7 @@ func (b *DockerGoBuilder) Build(ctx context.Context, in *api.BuildInput, output
 	// Write replace directives.
 	cmd := exec.CommandContext(ctx, "go", append([]string{"mod", "edit"}, replaces...)...)
 	cmd.Dir = plandst
-	_, err = cmd.CombinedOutput()
+	err = cmd.Run()
```

Comment:
This runs successfully, but if the goal is not to hide the command output, the output is still hidden like this.

To display the output you'd have to attach a buffer to stdout and stderr, and then later print out those buffers.

I think the use of CombinedOutput is actually correct, just don't assign the bytes buffer to _.

```suggestion
	output, err := cmd.CombinedOutput()
```

Reply:
Ah, you're right. Go sends the output to /dev/null by default.